solutions/functions/modiji.js

```diff
@@ -0,0 +1,32 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+function calculateTotalFund(salaries, peopleCount){
+    let funds = 0;
+    for(let i = 0; i < peopleCount; i++){
+        funds += Math.ceil(0.07 * salaries[i]);
```

Comment:
hi @Abduliitr 
if we are calculating the 7% of salary using the approach -> 0.07 * salaries[i], for sample salary of 100 we are getting the output as 7.0000001, which is incorrect and making ceil of this output we are getting the final output as 8 which is +1 of the correct output
but if we calculate using (7*salaries[i])/100) we are getting the correct output , and also even if we use pen and paper for the sample test case provided in problem description the output  should be 109, try to modify the code with the appropriate approach and commit it

Reply:
Updated

`        funds += Math.ceil((7*salaries[i])/100);
`